wger/software/static/css/agency.css

```diff
@@ -582,7 +582,6 @@ aside.clients img {
 }
 section#contact {
   background-color: #222222;
-  background-image: url('../img/map-image.png');
```

Comment:
The reference was to an image that could not be found on the system, I couldn't find an image with such name in the static folder